ktlint.gradle

```diff
@@ -0,0 +1,25 @@
+repositories {
+    jcenter()
+}
+
+configurations {
+    ktlint
+}
+
+dependencies {
+    ktlint "com.pinterest:ktlint:0.35.0"
```

Comment:
You can use ktlint formatting rules with detekt-formatting plugin. https://github.com/arturbosch/detekt#adding-more-rule-sets. It includes ktlint jar and you can configure ktlint rules in detekt configuration file.